translation.rst

```diff
@@ -351,9 +351,7 @@ Translation Resource/File Names and Locations
 Symfony looks for message files (i.e. translations) in the following default locations:
 
 * the ``translations/`` directory;
-
-* the ``src/Resources/<bundle name>/translations/`` directory;
-
+* the ``translations/<bundle name>/`` directory;
```

Comment:
Is there any significance to this? What I mean is: of course `translations/<bundle name>/` is a valid path, as (I assume) that all files are loaded from the `translations/` directory recursively. So, is there anything special about this directory? Translations in this directory already override the ones from the bundle, so I wasn't sure.

Thanks!

Reply:
Yep, you're right, translation resources are overridden by domain, so I guess this line is useless, as well as [its definition](https://github.com/symfony/symfony/blob/020664e1efbf8a81dbcc0f7b9dcbfa9ac58c0ed7/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1157-L1159).

Reply:
Removing useless definition https://github.com/symfony/symfony/pull/25476 in core.